docs/modules/sql/pages/user-defined-types.adoc

```diff
@@ -1,60 +1,53 @@
 = User-Defined Types in SQL
 
-User-Defined Types (also known as UDTs, nested types, or nested fields) is an experimental feature that allows you to create
- custom data types that can be referenced in the xref:sql:create-mapping.adoc[CREATE MAPPING statement].
+User-Defined Types (also known as UDTs, nested types, or nested fields) allow you to create custom data types that can be referenced in the xref:sql:create-mapping.adoc[CREATE MAPPING statement].
 
-UDTs are useful for creating and accessing hierarchical data structures, including simple cases of nested objects and
-more complex cases of fully/partially connected graphs of objects. See <<queryingExamples,examples>> for more information.
-
-WARNING: Support for UDTs is an experimental feature, and disabled by default.
-See <<enabling-udt-support, Enabling UDT support>> to learn how to enable it.
+UDTs are useful for creating and accessing hierarchical data structures, which can even be cyclic. See <<queryingExamples,examples>> for more information.
 
 == Feature Overview
 
-Due to the experimental nature of UDTs, the feature set is limited to the following:
+- All serialization formats (namely `portable`, `compact`, `java` and `avro`) are supported.
```

Comment:
mapping format != serialization format. In particular Avro is not HZ serialization format, DS/IDS is not explicitly configured in SQL.

Reply:
BTW, we don't support JSON for UDTs, do we? (it is supported as MAPPING format)

Reply:
I removed the "serialization" word. (099f756d) We don't support JSON for UDTs.